Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/compute_normal.h

```diff
@@ -266,7 +266,10 @@ compute_vertex_normal(typename boost::graph_traits<PolygonMesh>::vertex_descript
     he = opposite(next(he, pmesh), pmesh);
   } while (he != end);
 
-  if ( ! traits.equal_3_object()(normal, CGAL::NULL_VECTOR))
+  if(normal.x() != normal.x()
```

Comment:
You are normalizing only if one coord is a NAN?

Reply:
Arf I thought I removed that.

Reply:
It is acually absolutely not the right commit